demo-app/src/POCMap.jsx

```diff
@@ -3,14 +3,66 @@ import { Box } from 'grommet';
 import { Grommet } from 'grommet-icons';
 import {
   Controls,
+  Cluster,
   Map,
   Marker,
   MarkerCluster,
   Pin,
 } from 'grommet-leaflet-core';
-
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusWarningSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
 import { generateLocations, userLocation } from './utils/locations';
 import { ClusterPopup } from './ClusterPopup';
+import { getClusterStatus, getClusterSize } from './utils/status';
```

Comment:
For later scope, the size aspect seems like valuable functionality to ship with Cluster by default and not have to be implemented by caller.

Caller would be able to consume defaults, specify via props, and specify by theme.

Reply:
That's fair. I'll leave as is for now and we can incorporate it but in later on. We should think about how the caller can flag their desired thresholds to step from one size to the next (does that live in the theme, as a prop?). Depending on the use case, that step might differ.